### ISSUE TITLE
Accelerometer and gyroscope issue on Linux

### ISSUE THREAD

**Issue by Scall** [libretro/RetroArch#15166]
Even though the “Auxiliary Sensor Input” option is enabled, the accelerometer of a DualShock 4 can't be used in “Kirby Tilt 'n' Tumble” and the gyroscope in "WarioWare Twisted" can't either, when using the mGBA core in RetroArch 1.15.0 on Linux (Gentoo, kernel 6.2.10, hid-playstation driver). Both work without issues with the **standalone** emulator, but they don't with both the libretro core from the mgba-emu/mgba repository (version 0.10.1) and the one from the libretro/mgba repository (latest commit). I've been told by the mGBA developer that the core is properly hooked up to receive the required information, so it should be a RetroArch issue.

**Comment by gouchi** [comment]
Hi @Scall,

it should work now using SDL2 input/joypad driver using stable version.

I confirm it is working using DS4 on Linux with mGBA Libretro core.

Thank you.

**Comment by Scall** [comment]
Yes, with `input_joypad_driver = "sdl2"` it now works in both the games I mentioned, using the DualShock 4. But with a difference compared to mGBA standalone: using the RetroArch mGBA core, in “Kirby Tilt 'n' Tumble”, to go **to the left** or **to the right** I have to tilt the controller **in the opposite direction**. I guess this is an issue of the core, though.

Thank you, @gouchi.

**Comment by gouchi** [comment]
@Scall Thank you for the feedback.

Please close this issue as it is fixed using SDL2 driver ;)
